Gemfile.lock

```diff
@@ -581,3 +568,5 @@ DEPENDENCIES
 RUBY VERSION
    ruby 2.7.0p0
 
+BUNDLED WITH
+   2.1.2
```

Comment:
Seems like I've bundled with bundler v2. Is this going to cause issues for those not using bundle 2?

Reply:
I think it will just give them an annoying message.